.github/workflows/update-specs-and-client-libraries.yaml

```diff
@@ -158,6 +158,7 @@ jobs:
           - generator: java/okhttp-gson
             git_repo_id: onfido-java
             preCommit: |
+              sed -i "s/ *$//" pom.xml
```

Comment:
this is in the java generator but part of python MR? is this intended?

Reply:
yeah, that's correct. I didn't say specifically but I've considered that just as improvement of CI. It's to avoid that remaining trailing spaces introduced by the comments remain after code generation, causing differences since code editors tend to remove them.